doc/classes/ProjectSettings.xml

```diff
@@ -2028,14 +2133,19 @@
 			Scales the 3D render buffer based on the viewport size uses an image filter specified in [member rendering/scaling_3d/mode] to scale the output image to the full viewport size. Values lower than [code]1.0[/code] can be used to speed up 3D rendering at the cost of quality (undersampling). Values greater than [code]1.0[/code] are only valid for bilinear mode and can be used to improve 3D rendering quality at a high performance cost (supersampling). See also [member rendering/anti_aliasing/quality/msaa_3d] for multi-sample antialiasing, which is significantly cheaper but only smooths the edges of polygons.
 		</member>
 		<member name="rendering/shader_compiler/shader_cache/compress" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], compiled shaders that are cached to disk will be stored in a compressed format.
 		</member>
 		<member name="rendering/shader_compiler/shader_cache/enabled" type="bool" setter="" getter="" default="true">
+			Enable the shader cache, which stores compiled shaders to disk to prevent stuttering from shader compilation the next time the shader is needed.
 		</member>
 		<member name="rendering/shader_compiler/shader_cache/strip_debug" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], debug symbols will not be included in cached shader files, saving disk space.
```

Comment:
I don't know if there's any other benefit like a performance improvement, if someone knows that would be good to add also.

Reply:
It looks like the variable this sets is never even used. https://github.com/godotengine/godot/blob/432b25d3649319517827dbf7bc275e81e0a2b92e/servers/rendering/renderer_rd/shader_rd.cpp#L689-L696

Might be better to remove the description for now